tap-gui/tap-gui-rbac/assigning-kubernetes-roles.md

```diff
@@ -0,0 +1,125 @@
+# Assigning Roles and Permissions on Kubernetes Clusters
+
+This section provides a basic overview of how to create roles and persmissions on Kubernetes clusters and how to assign these roles to users. For more detailed information, please refer to [Using RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
```

Comment:
It's definitely nice to give an at-a-glance reference so folks can get started without linking somewhere else, but I'm a bit worried about duplicating (with slight changes in tone/emphasis) the upstream k8s docs. Just a nit, though, especially if @RichardJJG is into this I don't feel that strongly.

Reply:
It's a tricky one because none of this feels specific to TAP GUI. If this all just standard K8s stuff, maybe we should just link to Using RBAC Authorization and let the reader learn from the source.
On the other hand...that K8s doc is really long and is almost more reference topic than procedure. I can see why a customer would like this topic as a means to orientate himself (this is assuming the customer isn't well-versed in K8s).

Reply:
No-one has weighed in, so I'm marking this as resolved and leaving the content in.